File: Swift-C++-IR-Comparison/test/collectionTypes/Swift/Arrays/ConstantArray.swift
@main
struct MyMain {

  static func main() {
    let emptyArrayNoType: [Any] = []
    let emptyArray: [Int] = []

    let numberArrayNoType = [1, 2, 3, 4, 5]
    let numberArrayExplicitType: [Int] = [1, 2, 3, 4, 5]

    let stringArrayNoType = ["Test", "Input"]
    let stringArrayExplicitType: [String] = ["Test", "Input"]
  }
}

File: Swift-C++-Testsuite/test/collectionTypes/C++/Arrays/ConstantArray.cpp

#include <array>
#include <string>
int main()
{
    std::array<void*, 0> emptyArrayNoType; // in Swift this is an Any type
    std::array<int, 0> emptyArray;  

    int intArray[5] = {1, 2, 3, 4, 5};  // Create an array of integers with initial values

    std::array<std::string, 2> stringArray = {"Test", "Input"};  // Create an array of strings with initial values
}
